vxdetector/Output_counter.py

```diff
@@ -18,58 +22,120 @@ def directory_navi(file_name, path, dir_name, dir_path):
     return file_name, dir_name, dir_path
 
 
-def create_output(path, file_name, unaligned_count, most_probable_V,
-                  probability, dir_name, dir_path):
+def rawincount(filename):
+    f = open(filename, 'rb')
+    bufgen = takewhile(lambda x: x, (f.raw.read(1024*1024)
+                                     for _ in repeat(None)))
+
+    return sum(buf.count(b'\n') for buf in bufgen)
+
+
+def region_count(dictionary, unaligned_count, temp_path, all_reads, mode):
+    if temp_path is not None and mode == 'paired':
+        unpaired = round((rawincount(f'{temp_path}bed.log')/all_reads), 2)
+    elif mode == 'unpaired':
+        unpaired = 'unpaired Reads'
+    no_V = rawincount(f'{temp_path}noOver.bed')
+    aligned_count = 100 - unaligned_count
+    count = sum([dictionary['V1'], dictionary['V2'], dictionary['V3'],
+                dictionary['V4'], dictionary['V5'], dictionary['V6'],
+                dictionary['V7'], dictionary['V8'], dictionary['V9'], no_V])
+    dictionary['V1'] = round((dictionary['V1'] / count) * aligned_count, 2)
+    dictionary['V2'] = round((dictionary['V2'] / count) * aligned_count, 2)
+    dictionary['V3'] = round((dictionary['V3'] / count) * aligned_count, 2)
+    dictionary['V4'] = round((dictionary['V4'] / count) * aligned_count, 2)
+    dictionary['V5'] = round((dictionary['V5'] / count) * aligned_count, 2)
+    dictionary['V6'] = round((dictionary['V6'] / count) * aligned_count, 2)
+    dictionary['V7'] = round((dictionary['V7'] / count) * aligned_count, 2)
+    dictionary['V8'] = round((dictionary['V8'] / count) * aligned_count, 2)
+    dictionary['V9'] = round((dictionary['V9'] / count) * aligned_count, 2)
+    no_V = round((no_V/count)*aligned_count, 2)
+    if aligned_count == 0 or (no_V/aligned_count) == 1:
+        most_probable_V = 'No variable Region'
+    else:
+        most_probable_V = [x[0].replace('V', '') for x in
+                           sorted(list(dictionary.items()))
+                           if ((x[1] / aligned_count) * 100) > 20]
+        most_probable_V = 'V' + ''.join(most_probable_V)
+        if most_probable_V == 'V':
+            most_probable_V = 'No variable Region'
+
+    return most_probable_V, dictionary, unpaired, no_V
+
+
+def create_output(path, file_name, unaligned_count, dictionary, dir_name,
```

Comment:
have a look into `pandas.to_csv` to `pandas.to_excel`. That might ease your life significantly!